packages/plugins/src/sample-project/sample-foundry-config.ts

```diff
@@ -54,7 +54,12 @@ dist/
 
 export const fetchDotEnvFile = (
   sphinxApiKey: string,
+  sphinxOrgId: string,
   alchemyApiKey: string
 ): string => {
-  return `SPHINX_API_KEY=${sphinxApiKey}\n` + `RPC_API_KEY=${alchemyApiKey}`
+  return (
+    `SPHINX_API_KEY=${sphinxApiKey}\n` +
+    `SPHINX_ORG_ID=${sphinxOrgId}` +
+    `RPC_API_KEY=${alchemyApiKey}`
+  )
```

Comment:
The demo test suite doesn't use these values because it just deploys using the deploy CLI command. I've updated the test check that they were written correctly.